main.c

```diff
@@ -127,7 +127,7 @@ static int searchKeyNum(void);
 #define help() fusage(stdout, 0)
 #define usage() fusage(stderr, 1)
 
-int enable_inline_image;
+int enable_inline_image = -1; /* TODO: can't we remove this ifndef USE_IMAGE? */
```

Comment:
I don't see an USE_IMAGE here and probably no ;)

Reply:
I think it's unused without USE_IMAGE, but you also have to ifdef out a few enable_inline_image checks (and the CLI param assignments).